RemoteHealthcare/ConsoleGUI.cs

```diff
@@ -1,111 +1,149 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Text;
 using System.Threading.Tasks;
 
 namespace RemoteHealthcare
 {
+    enum MenuOptions
+    {
+        START_SIM = '0',
+        RUN_STEP = '1',
+        READ_BIKE = '2',
+        READ_HEARTHRATE = '3',
+        CLOSE_CONNECTION = '4',
+    }
+
     class ConsoleGUI
     {
-        public Task SelectionHandler(Program program)
+        readonly Program program;
+
+        public ConsoleGUI(Program program)
+        {
+            this.program = program;
+        }
+
+        public Task SelectionHandler()
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#methodes) voor fout bij Methode structuur fout (Danwel hoe methode is opgebouwd of commentaar)